jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,74 @@
+import boto3
+import sys
+
+
+def main():
+    try:
+        # connect to aws lambda
+        client = boto3.client('lambda')
+    except Exception, e:
+        # if connection failed, raise error
+        raise ValueError(e)
+        sys.exit(0)
+
+    # ensure the number of arguments is correct
+    num_args = len(sys.argv)
+
+    if num_args == 2:
```

Comment:
Sorry! Accidentally screwed up the else statement indentation while fixing style changes